### PR TITLE
Update index.mdx

### PR DESCRIPTION
**Pull request by BritneyMuller**
# What does this PR do?

This PR updates our Expert Acceleration Program image with a new image featuring our experts.

This is similar to our [Transformers/README.md image update](https://github.com/huggingface/transformers/pull/16615) that has proven to be successful.

support-image-update

### CODE REVIEW

File: docs/source/en/index.mdx
Comment (sgugger) [inline docs/source/en/index.mdx:31]:
```suggestion
<img alt="HuggingFace Expert Acceleration Program" src="https://cdn-media.huggingface.co/marketing/transformers/new-support-improved.png" style="max-width: 600px; border: 1px solid #eee; border-radius: 4px; box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.05);">
```
Don't think we need the extra space here.

**Review by sgugger — APPROVED** [review]
Thanks a lot for switching!

**Review by feconroses — APPROVED** [review]
Great idea. LGTM! Let's wait for someone from the open source team to review+approve before merging.

**Pull request merged**
